### ISSUE TITLE
[Bug]: RSS Feeds not respecting "Prevent feed from being indexed by itunes and google" 

### ISSUE THREAD

**Issue by DieselTech** [advplyr/audiobookshelf#2047]
### Describe the issue

Turning on Prevent feed from being indexed by itunes and google doesn't apply the protections it says it does. This is confirmed both by the RSS status window showing "Prevent Indexing: No" and by checking the RSS feed itself. 

```rss
<itunes:block>No</itunes:block>
<googleplay:block>no</googleplay:block>
```

### Steps to reproduce the issue

1. Turn on a feed
2. Check the box to prevent
3. View Feed

### Audiobookshelf version

2.3.3

### How are you running audiobookshelf?

Docker

**Comment by advplyr** [comment]
I wasn't able to reproduce this in my testing. With the prevent indexing checked I get:

```xml
<itunes:block>Yes</itunes:block>
<googleplay:block>yes</googleplay:block>
```

I did some reading on this though and made minor updates. I updated them to use lowercase "yes" and when not preventing indexing those tags won't be present at all. Using "no" in that tag is useless.

Are you still able to reproduce this on 2.3.3?

**Comment by DieselTech** [comment]
> Are you still able to reproduce this on 2.3.3?

Yes. I even turned off existing feeds that were made on prior versions and recreated them on 2.3.3

The behavior is 100% reproducible on my instance using the latest docker container.

**Comment by advplyr** [comment]
Are you able to get it to say "Yes" here at all?

![image](https://github.com/advplyr/audiobookshelf/assets/67830747/663206fa-f138-4cd5-9a04-3fdb06bf4144)

**Comment by DieselTech** [comment]
> Are you able to get it to say "Yes" here at all?

Nope. No combo of actions I could take would ever get it to show up as `Yes`

**Comment by nichwall** [comment]
I was just able to recreate this using a collection on 5c1c5117180765d32d325b7ab682bf31addfabc6 (edge on October 27 2023, most recent release was 2.4.4). When opening an RSS feed from a single book the Prevent Indexing was set correctly, but was not set for a collection with two books.
![image](https://github.com/advplyr/audiobookshelf/assets/5686638/62aca0dc-2e1f-4b80-8da9-739812cc083d)

**Comment by advplyr** [comment]
Thanks, I found it. It was specific to collections & series

**Comment by advplyr** [comment]
Fixed in [v2.5.0](https://github.com/advplyr/audiobookshelf/releases/tag/v2.5.0)
